osu.Game/Overlays/Profile/Header/Components/DailyChallengeStreakTooltip.cs

```diff
@@ -0,0 +1,251 @@
+using osu.Game.Graphics.Sprites;
+using osu.Game.Online.API.Requests.Responses;
+using osu.Game.Scoring;
+using osuTK;
+using Box = osu.Framework.Graphics.Shapes.Box;
+using Color4 = osuTK.Graphics.Color4;
+
+namespace osu.Game.Overlays.Profile.Header.Components
+{
+    public partial class DailyChallengeStreakTooltip : VisibilityContainer, ITooltip<DailyChallengeStreakTooltipData>
+    {
+        private StreakPiece currentDaily = null!;
+        private StreakPiece currentWeekly = null!;
+        private StatisticsPiece bestDaily = null!;
+        private StatisticsPiece bestWeekly = null!;
+        private StatisticsPiece topTen = null!;
+        private StatisticsPiece topFifty = null!;
+
+        private Box topBackground = null!;
+        private Box background = null!;
+
+        [Resolved]
+        private OsuColour colours { get; set; } = null!;
+
+        [BackgroundDependencyLoader]
+        private void load()
+        {
+            AutoSizeAxes = Axes.Both;
+            CornerRadius = 20f;
+            Masking = true;
+
+            EdgeEffect = new EdgeEffectParameters
+            {
+                Type = EdgeEffectType.Shadow,
+                Colour = Color4.Black.Opacity(0.5f),
+                Radius = 30f,
+            };
+
+            Children = new Drawable[]
+            {
+                background = new Box
+                {
+                    RelativeSizeAxes = Axes.Both,
+                },
+                new FillFlowContainer
+                {
+                    AutoSizeAxes = Axes.Both,
+                    Direction = FillDirection.Vertical,
+                    Children = new Drawable[]
+                    {
+                        new Container
+                        {
+                            AutoSizeAxes = Axes.Both,
+                            Children = new Drawable[]
+                            {
+                                topBackground = new Box
+                                {
+                                    RelativeSizeAxes = Axes.Both,
+                                },
+                                new FillFlowContainer
+                                {
+                                    AutoSizeAxes = Axes.Both,
+                                    Direction = FillDirection.Horizontal,
+                                    Padding = new MarginPadding(15f),
+                                    Spacing = new Vector2(30f),
+                                    Children = new[]
+                                    {
+                                        // currentDaily = new StreakPiece(UsersStrings.ShowDailyChallengeDailyStreakCurrent),
+                                        // currentWeekly = new StreakPiece(UsersStrings.ShowDailyChallengeWeeklyStreakCurrent),
+                                        currentDaily = new StreakPiece("Current Daily Streak"),
+                                        currentWeekly = new StreakPiece("Current Weekly Streak"),
```

Comment:
What's this?

Reply:
Pending resources-side localisation update to use the ones commented, meant to add a task for that.

Reply:
Have applied these.